python/pyarrow/tests/test_dataset.py

```diff
@@ -2672,47 +2672,57 @@ def test_feather_format(tempdir, dataset_reader):
         dataset_reader.to_table(ds.dataset(basedir, format="feather"))
 
 
-def _create_parquet_dataset_simple(root_path):
+def _create_parquet_dataset_simple(root_path, use_legacy_dataset=True):
```

Comment:
```suggestion
def _create_parquet_dataset_simple(root_path, use_legacy_dataset=True):
    """
    Creates a simple (flat files, no nested partitioning) Parquet dataset
    """
```

(to clarify the intent)

Reply:
Added.